src/app/component/ProductWishlistButton/ProductWishlistButton.container.js

```diff
@@ -170,6 +189,7 @@ export class ProductWishlistButtonContainer extends PureComponent {
         return (
             <ProductWishlistButton
               { ...this.props }
+              { ...this.state }
```

Comment:
I believe spreading state down the components has been prohibited by the latest ESLint config in order to reduce prop drilling capacity throughout the application.

Reply:
It is done like that everywhere in application. Then whole application needs to be refactored too I guess.